docs/docs/scheduled-triggers/recipes/check-expiration.mdx

```diff
@@ -296,8 +305,139 @@ nodemailer.createTestAccount((err, account) => {
 
 </details>
 
-You can run the server by running `node index.js` in your terminal. If you see the message
-`Webhook server is running on port 4000`, you're good to go!
+You can run the server by running `node index.js` in your terminal.
+
+</TabItem>
+<TabItem value="python">
+
+Make sure you have the necessary dependencies installed. You can use pip to install them:
+
+```bash
+pip install Flask secure-smtplib requests
+```
+
+<details>
+<summary>
+Then, create a new file called <code>index.py</code> and add the following code:
+</summary>
+
+```python
+from flask import Flask, request, jsonify
+import smtplib
+from smtplib import SMTPException
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+import requests
+from datetime import datetime, timedelta
+
+app = Flask(__name__)
+
+# Create an SMTP server connection
+def create_smtp_server():
+    try:
+        server = smtplib.SMTP(host='smtp.ethereal.email', port=587)
+        server.starttls()
+        server.login('example@superstore.com', 'samplePassword123')
+        return server
+    except SMTPException as e:
+        print(f"An error occurred while creating the SMTP server: {e}\nWe'll print the message to the terminal\n")
+        return None
+
+def get_expired_coupons():
+    today = datetime.now()
+    two_days_from_now = today + timedelta(days=2)
+    formatted_date = two_days_from_now.isoformat()
+
+    # Prepare the request headers
+    headers = {
+        'Content-Type': 'application/json',
+        'x-hasura-admin-secret': 'super_secret_string_123',
+    }
+
+    graphql_query = '''
+    query GetExpiringCoupons($two_days_from_now: timestamptz!) {
+      coupons_coupon(where: {expiration_date: {_lte: $two_days_from_now}}) {
+        id
+        expiration_date
+        code
+        user_data {
```

Comment:
```suggestion
        user {
```

In our schema, there is no `user_data` type. Please note that this will need to be updated throughout to match the sample app's schema.